backend/src/main/java/gov/cdc/usds/simplereport/SimpleReportApplication.java

```diff
@@ -60,6 +60,12 @@ public CommandLineRunner scheduleUploads(DataHubConfig config, ScheduledTasksSer
     return args -> scheduler.scheduleUploads(config);
   }
 
+  @Bean
+  @ConditionalOnProperty("simple-report.id-verification-reminders.enabled")
+  public CommandLineRunner scheduleAccountReminderEmails(ScheduledTasksService scheduler) {
+    return args -> scheduler.scheduleAccountReminderEmails("0 0 1 * * *", "America/New_York");
```

Comment:
I don't think we'll be changing these, so for now they're left hard-coded instead of in properties

Reply:
Yeah, I think this is fine for now!